src/message_protocols.ts

```diff
@@ -0,0 +1,7 @@
+// This file re-exports types for message protocols for background, content and commandline contexts
+
```

Comment:
This is done to safely export just the types. This won't translate to any runtime import, which is very important as we do not want to import content code in the background page, or vice versa.